packages/expo-updates/android/src/main/java/expo/modules/updates/loader/Loader.kt

```diff
@@ -313,7 +304,7 @@ abstract class Loader protected constructor(
         database.assetDao().insertAssets(finishedAssetList, updateEntity!!)
 
         if (erroredAssetList.size == 0) {
-          database.updateDao().markUpdateFinished(updateEntity!!, skippedAssetList.size != 0)
+          database.updateDao().markUpdateFinished(updateEntity!!, false)
```

Comment:
can we also get rid of this bool arg since it is always false?

Reply:
Yes, I added an `isEmbedded` flag to replace it.